bin/zopen-install

```diff
@@ -65,7 +79,8 @@ verifySignatureOfPax()
   # Verify the signature
   printInfo "Verifying the gpg signature..."
   if [ -f "$SIGNATURE_FILE" ]; then
-        if gpg --no-default-keyring --keyring "$TMP_GPG_DIR/pubring.kbx" --verify "$SIGNATURE_FILE" "$FILE_TO_VERIFY" 2>&1 | grep -q "Good signature from"; then
+	gpg_output=$(gpg --no-default-keyring --keyring "$TMP_GPG_DIR/pubring.kbx" --verify "$SIGNATURE_FILE" "$FILE_TO_VERIFY" 2>&1)
```

Comment:
Should it be echo'ing gpg_output in the fail case?

Reply:
Right! I have displayed the output in failure case. Thank you.